src/applications/gi/sass/partials/_gi-profile-page.scss

```diff
@@ -16,6 +16,10 @@
     margin: 0;
   }
 
+  .section-button {
+      letter-spacing: 0.4px !important;
```

Comment:
NIT: I'm not going to reject this PR over it, but `!important` should be avoided, if possible. 

> Using `!important`, however, is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets. - https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception

Reply:
Removing the `!important;` as css is still applied without it

This PR was not flagged to be reviewed by @department-of-veterans-affairs/frontend-review-group 

For context:
At 320px width the text is hidden behind the background image of the button
![image](https://user-images.githubusercontent.com/1094999/80734765-a2aeed80-8add-11ea-8815-035a17e453fe.png)

Using `letter-spacing: 0.4px;` fixes the issue, the accordion is using `usa-accordion` and other related classes to behave as an acccordion
![image](https://user-images.githubusercontent.com/1094999/80734784-a93d6500-8add-11ea-9147-8f25bcfeaaec.png)

This PR was not flagged to be review by @department-of-veterans-affairs/frontend-review-group